core/src/toga/platform.py

```diff
@@ -21,6 +21,7 @@
     "darwin": "macOS",
     "ios": "iOS",
     "linux": "linux",
+    "freebsd": "freeBSD",
```

Comment:
This isn't needed if the `startswith("freebsd")` change is in place. The android version is technically redundant right now, but there's a possibility that we're going to get `sys.platform` to return `android` in future, so the line is there for that future possibility.